src/app/public/testing/error-fixture.spec.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  Component,
+  Input
+} from '@angular/core';
+
+import {
+  TestBed
+} from '@angular/core/testing';
+
+import {
+  SkyErrorModule
+} from '@skyux/errors';
+
+import {
+  SkyErrorFixture
+} from './error-fixture';
+
+//#region Test component
+@Component({
+  selector: 'error-test',
+  template: `
+<sky-error [errorType]="errorType" data-sky-id="test-error"></sky-error>
+<sky-error data-sky-id="test-error-custom">
+  <sky-error-image>
+  </sky-error-image>
+</sky-error>
+  `
+})
+class TestComponent {
+
+  @Input()
+  public errorType = 'broken';
+
+}
+//#endregion Test component
+
+describe('Error fixture', () => {
```

Comment:
I noticed a new convention here with the approach to this testing the fixture instead of the component (`error-fixture.spec.ts` instead of `error.component.spec.ts`). Was this deliberate? Should we be approaching new tests this way?

Reply:
This is testing the `SkyErrorFixture` class, which is used by the consumer to assist in unit testing their own components (which use our components). I might be misunderstanding your question.

Reply:
That actually answers my question - I didn't understand the intent was for consumers. Makes total sense.